src/pharmpy/tools/amd/run.py

```diff
@@ -66,6 +66,7 @@ def run_amd(
     strictness: Optional[str] = "minimization_successful or (rounding_errors and sigdigs>=0.1)",
     dv_types: Optional[dict] = None,
     mechanistic_covariates: Optional[List[str]] = None,
+    perform_retries: Optional[str] = "amd_final",
```

Comment:
`perform_retries` sounds like a boolean. Perhaps only `retries` or `retries_strategy`.

Reply:
Changed to `retries_strategy`